<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportCsvDialog</class>
 <widget class="QDialog" name="ImportCsvDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import CSV file</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>&amp;Table name</string>
       </property>
       <property name="buddy">
        <cstring>editName</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelHeader">
       <property name="text">
        <string>&amp;Column names in first line</string>
       </property>
       <property name="buddy">
        <cstring>checkboxHeader</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkboxHeader">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelSeparator">
       <property name="text">
        <string>Field &amp;separator</string>
       </property>
       <property name="buddy">
        <cstring>comboSeparator</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboSeparator">
       <item>
        <property name="text">
         <string>,</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tab</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelQuote">
       <property name="text">
        <string>&amp;Quote character</string>
       </property>
       <property name="buddy">
        <cstring>comboQuote</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboQuote">
       <item>
        <property name="text">
         <string>&quot;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>'</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tablePreview"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editName</tabstop>
  <tabstop>checkboxHeader</tabstop>
  <tabstop>comboSeparator</tabstop>
  <tabstop>comboQuote</tabstop>
  <tabstop>tablePreview</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImportCsvDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImportCsvDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkboxHeader</sender>
   <signal>toggled(bool)</signal>
   <receiver>ImportCsvDialog</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboSeparator</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ImportCsvDialog</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboQuote</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ImportCsvDialog</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ImportCsvDialog</receiver>
   <slot>checkInput()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>609</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>631</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updatePreview()</slot>
  <slot>checkInput()</slot>
 </slots>
</ui>
